oci/runtime_oci.go

```diff
@@ -305,6 +309,13 @@ func (r *runtimeOCI) ExecContainer(c *Container, cmd []string, stdin io.Reader,
 		}
 
 		cmdErr = execCmd.Run()
+
+		// The read side of the pipe should be closed after the container process has been started.
```

Comment:
This looks incorrect. Did we miss the start/wait split here?

Reply:
ah, oops, we sure did

Reply:
fixed!